### ISSUE TITLE
Unable to Scan Organization Folder. /computation says it succeeds but logs show NPE

### ISSUE THREAD

**Issue by gbloggs** [jenkinsci/bitbucket-branch-source-plugin#395]
### Your checklist for this issue

- [x] Jenkins version

- [x] Plugin version

- [x] Bitbucket cloud

- [x] Bitbucket server and version

### Description

Jenkins: 	2.264
bitbucket-branch-source-plugin: 2.9.4
branch-api: 2.6.0
Bitbucket Cloud: yes
Bitbucket Server: no

I am trying to configure a completely new Jenkins build against Bitbucket cloud. Yesterday we did manage to get the scanning working (although the automatic webhook configuration did not appear to work). Today, after upgrading Jenkins and all the plugins that it identified as needing updating, branch scanning no longer works.

Looking at the logs it seems this line is ultimately causing a Null Pointer:

`at com.cloudbees.jenkins.plugins.bitbucket.BitbucketSCMNavigator.visitSources(BitbucketSCMNavigator.java:501)`

The full log from this error:
```
 1 branches were processed (query completed)
ERROR: Failed to create or update a subproject my-project
java.lang.NullPointerException
	at jenkins.branch.OrganizationChildOrphanedItemsProperty.decorate(OrganizationChildOrphanedItemsProperty.java:108)
	at jenkins.branch.OrganizationFolderProperty.applyDecoration(OrganizationFolderProperty.java:54)
	at jenkins.branch.OrganizationFolder$SCMSourceObserverImpl$1.completeNew(OrganizationFolder.java:1573)
	at jenkins.branch.OrganizationFolder$SCMSourceObserverImpl$1.complete(OrganizationFolder.java:1504)
	at jenkins.scm.api.trait.SCMNavigatorRequest.process(SCMNavigatorRequest.java:254)
	at jenkins.scm.api.trait.SCMNavigatorRequest.process(SCMNavigatorRequest.java:204)
	at com.cloudbees.jenkins.plugins.bitbucket.BitbucketSCMNavigator.visitSources(BitbucketSCMNavigator.java:501)
	at jenkins.branch.OrganizationFolder.computeChildren(OrganizationFolder.java:542)
	at com.cloudbees.hudson.plugins.folder.computed.ComputedFolder.updateChildren(ComputedFolder.java:278)
	at com.cloudbees.hudson.plugins.folder.computed.FolderComputation.run(FolderComputation.java:165)
	at jenkins.branch.OrganizationFolder$OrganizationScan.run(OrganizationFolder.java:1033)
	at hudson.model.ResourceController.execute(ResourceController.java:97)
	at hudson.model.Executor.run(Executor.java:429)
```

**Comment by gbloggs** [comment]
I can confirm that if Jenkins is rolled back to 2.263 this issue is no longer seen, so ppears to be an isse with the 2.264 build as this was the only thing I rolled back.

**Comment by peterdporter** [comment]
I am also seeing this issue

**Comment by joshadambell** [comment]
Same issue on 2.265

**Comment by gravcat** [comment]
Also verified this problem on 2.265 (newest "weekly" as of writing)

```
Looking up reallycoolorganization/new-project for branches
Checking branch master from reallycoolorganization/new-project
      ‘Jenkinsfile’ found
    Met criteria

  1 branches were processed (query completed)
ERROR: Failed to create or update a subproject new-project
java.lang.NullPointerException
	at jenkins.branch.OrganizationChildOrphanedItemsProperty.decorate(OrganizationChildOrphanedItemsProperty.java:108)
	at jenkins.branch.OrganizationFolderProperty.applyDecoration(OrganizationFolderProperty.java:54)
	at jenkins.branch.OrganizationFolder$SCMSourceObserverImpl$1.completeNew(OrganizationFolder.java:1573)
	at jenkins.branch.OrganizationFolder$SCMSourceObserverImpl$1.complete(OrganizationFolder.java:1504)
	at jenkins.scm.api.trait.SCMNavigatorRequest.process(SCMNavigatorRequest.java:254)
	at jenkins.scm.api.trait.SCMNavigatorRequest.process(SCMNavigatorRequest.java:204)
	at com.cloudbees.jenkins.plugins.bitbucket.BitbucketSCMNavigator.visitSources(BitbucketSCMNavigator.java:501)
	at jenkins.branch.OrganizationFolder.computeChildren(OrganizationFolder.java:542)
	at com.cloudbees.hudson.plugins.folder.computed.ComputedFolder.updateChildren(ComputedFolder.java:278)
	at com.cloudbees.hudson.plugins.folder.computed.FolderComputation.run(FolderComputation.java:165)
	at jenkins.branch.OrganizationFolder$OrganizationScan.run(OrganizationFolder.java:1033)
	at hudson.model.ResourceController.execute(ResourceController.java:97)
	at hudson.model.Executor.run(Executor.java:429)
```

Configuring this on a fresh install of 2.249.3 (newest stable at time of writing) however, works as expected

```
Proposing reallycoolorganization/new-project
Connecting to https://bitbucket.org using somecreds/******** (my-credentials)
Repository type: Git
Looking up reallycoolorganization/new-project for branches
Checking branch master from reallycoolorganization/new-project
      ‘Jenkinsfile’ found
    Met criteria

  1 branches were processed (query completed)
```

**Comment by aferreeira** [comment]
I can confirm that version 2.269 has solved this problem

**Comment by lurume84** [comment]
Issue was solved for me after I installed branch-api 2.6.6 update. I had 2.5.0 and OP had 2.6.0 so the issue may be solved at branch-api >2.6.0.

Jenkins: 2.289.2
bitbucket-branch-source-plugin: 2.9.9
branch-api: 2.6.6
Bitbucket Cloud: yes
Bitbucket Server: no
